### PR TITLE
PRM-2975 - R5B code removed

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

File: it/controllers/PenaltiesFrontendControllerISpec.scala [hunk 1]
```diff
@@ -452,8 +451,8 @@ class PenaltiesFrontendControllerISpec extends IntegrationSpecCommonBase with ET
       Json.parse(result.body) shouldBe combinedPenaltyAndFinancialData
     }
 
-    "the get penalty details call includes blank expiryReason and appealLevel fields" in {
-      mockStubResponseForGetPenaltyDetails(Status.OK, "123456789", body = Some(getPenaltyDetailsJsonWithBlankExpiryReasonAndAppealLevel.toString()))
+    "the get penalty details call includes blank appealLevel fields" in {
```
Comment (LewisMitchell) [inline it/controllers/PenaltiesFrontendControllerISpec.scala:454]: Not sure this is a valid test anymore, given the above?

**Comment by AndrewRButcher** [comment]
app config prod PR https://github.com/hmrc/app-config-production/pull/19993

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: it/controllers/PenaltiesFrontendControllerISpec.scala [hunk 2]
```diff
@@ -93,7 +93,7 @@ class PenaltiesFrontendControllerISpec extends IntegrationSpecCommonBase with ET
       |}
       |""".stripMargin)
 
-  val getPenaltyDetailsJsonWithBlankExpiryReasonAndAppealLevel: JsValue = Json.parse(
+  val getPenaltyDetailsJsonWithBlankAppealLevel: JsValue = Json.parse(
```
[the remark this answers is not in the document]
Comment (AndrewRButcher) [reply it/controllers/PenaltiesFrontendControllerISpec.scala:96]: I think there is case where it wont be populated, when the penalty is unappealable as for if this is handled via being optional I am not confident hence I left this as is. If we are sure it is not needed anymore I am happy to remove

[the remark this answers is not in the document]
Comment (LewisMitchell) [reply it/controllers/PenaltiesFrontendControllerISpec.scala:96]: Ah, sorry. I understand now, keep it. Thanks

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**